config/routes.rb

```diff
@@ -2,7 +2,13 @@
 
 Rails.application.routes.draw do
   resources :kudos
-  devise_for :employees
+  devise_for :admin_users, path: 'admin'
+  devise_for :employees, path: 'employees'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   root 'kudos#index'
+  get '/admin/', to: 'admin#dashboard'
+
+  namespace :admin do
+    root to: 'admin#dashboard'
```

Comment:
Which controller, and which action will be called here?

Reply:
i thought that `AdminController` with action `dashboard` (that doesn't exist).
I think right now i understand how namespaces and routes work. Can you check again

Reply:
Great :) So the last question is: why it worked with incorrect routes, and actions in the first place? (it could be really nice TIL btw :)